src/renderers/webxr/WebXRManager.js

```diff
@@ -275,7 +275,11 @@ class WebXRManager extends EventDispatcher {
 				currentPixelRatio = renderer.getPixelRatio();
 				renderer.getSize( currentSize );
 
-				const useLayers = session.enabledFeatures !== undefined && session.enabledFeatures.includes( 'layers' );
+				// Check that the browser implements all the necessary APIs to use an
+				// XRProjectionLayer rather than an XRWebGLLayer
+				const useLayers = XRWebGLBinding !== undefined &&
+													'createProjectionLayer' in XRWebGLBinding.prototype &&
+													session.renderState.layers !== undefined;
```

Comment:
do we still need to check for 'layers'? It seems every browser that has `createProjectionLayer` will have that entry.

Reply:
Probably not, no! I can remove that.

Reply:
Done. And I've verified that this fix is working as intended on the Quest browser.